### PR TITLE
Update to allow setting from the command line

### PR DESCRIPTION
**Pull request by JamesJeffryes**

### CODE REVIEW

File: productivity/driver.py [hunk 1]
```diff
@@ -201,7 +201,7 @@ def get_tags(self):
             else:
                 # Empty modbus addresses or odd length strings could land you on a
                 # register that's not used
-                decoder._pointer += 1
+                decoder._pointer += 2
```
Comment (patrickfuller) [inline productivity/driver.py]: Seems outside the scope of the PR.

File: productivity/__init__.py [hunk 2]
```diff
@@ -12,19 +12,28 @@ def command_line():
...
     parser.add_argument('address', help="The IP address of the PLC.")
     parser.add_argument('tags', help="The PLC tag database file.")
+    parser.add_argument('-s', '--set', type=yaml.load,
+                        help="Pass a YAML string with parameters to be set.")
     args = parser.parse_args()
 
     async def get():
         async with ProductivityPLC(args.address, args.tags) as plc:
             d = await plc.get()
             print(json.dumps(d, indent=4))
```
Comment (patrickfuller) [inline productivity/__init__.py:30]: How about simplifying to one function - feels cleaner to me.

```python
async def run():
        async with ProductivityPLC(args.address, args.tags) as plc:
            if args.set:
                await plc.set(**args.set)
            d = await plc.get()
            print(json.dumps(d, indent=4))
loop.run_until_complete(run())
```

File: README.md [hunk 3]
```diff
@@ -32,10 +32,18 @@ you don't need to remember the addresses.
 
 ### Command Line
 
+To print the tags and their values, simply call the library with the PLC IP address and 
+the tags file.
 ```
 $ productivity the-plc-ip-address path/to/tags.csv
 ```
 
+Set values on the PLC from the command line using the --set flag with a string of valid
```
Comment (patrickfuller) [inline README.md]: I like the API but leading with "valid YAML" sounds complicated. How about:

---

Use `--set` to set values on the PLC:

```
$ productivity the-plc-ip-address path/to/tags.csv --set "{valve: false, setpoint: 4.45, label: foo}"
```

The `--set` parameter takes [YAML](https://en.wikipedia.org/wiki/YAML), a simple data format that allows you to easily set multiple tags at once.

---

The idea here is to allow the user to pattern-match for themselves before being introduced to new terminology (which could take them down an unnecessarily complicated path). This version also gives a soft intro, which might provide enough explanation for users to avoid going down a googling path.

**Review by patrickfuller — CHANGES_REQUESTED** [review]
Minor changes around reducing barriers to entry for super beginners.

**Comment by JamesJeffryes** [comment]
Closes #3
